### DISCUSSION TITLE
EQ6 reflections

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/library#293] [General]
I forgot to do this soon after class so I don't remember as well. 🤦‍♀️ It went fairly smooth though.

One thing I remember though is that it might help to start with an activity that refers back to solving liner equations that have fractions. How we can tackle those by multiplying through by a carefully chosen number to get the denominators to cancel out. Bridge that to what we're doing now, which uses expressions instead of just numbers.

**Comment by siwelwerd** [comment]
See #564
